generator/internal/service_code_generator.h

```diff
@@ -185,6 +185,12 @@ class ServiceCodeGenerator : public GeneratorInterface {
    */
   bool HasGenerateGrpcTransport() const;
 
+  /**
+   * Determines if protobuf to JSON transcoding should preserve field names as
+   * they appear in the proto definition or render the field names in camelCase.
+   */
+  bool HasPreserveProtoFieldNamesInJson() const;
+
```

Comment:
My `Ctrl`+`F` might be failing me, but it looks like this is unused. Remove?

And if it can't be removed, I would s/HasPreserve/Preserves/